app/dashboard/views.py

```diff
@@ -1202,22 +1202,25 @@ def profile(request, handle):
 @csrf_exempt
 def lazy_load_kudos(request):
     page = request.POST.get('page', 1)
-    address = request.POST.get('address')
     context = {}
     datarequest = request.POST.get('request')
     order_by = request.GET.get('order_by', '-modified_on')
```

Comment:
looks like the order was removed from the front end so if we are not going to have it there is not need to have it in the request

Reply:
I'll remove it in a follow up. Thanks for the heads up!